src/item_pocket.cpp

```diff
@@ -1321,6 +1322,15 @@ bool item_pocket::airtight() const
     return data->airtight;
 }
 
+bool item_pocket::allows_speedloader( const itype_id &speedloader_id ) const
+{
+    if( data->allowed_speedloaders.empty() ) {
+        return true;
```

Comment:
Shouln't it be `false`?
```suggestion
        return false;
```

Reply:
i did do that on purpose but i'm not exactly married to that decision

Reply:
What is the reason to make it `true`? If a pocket does not allow any speedloaders at all, why would it allow a specific speedloader?

Reply:
For example, with this logic, it appears that I can reload my COP .357 4-round Derringer with a 5-round, 6-round, or 7-round speedloader:

![image](https://user-images.githubusercontent.com/238395/93635424-7bcf5600-f9af-11ea-8343-d1d278008ed7.png)